sdk/servicebus/service-bus/test/public/amqpAnnotatedMessage.spec.ts

```diff
@@ -3,138 +3,283 @@
 
 import chai from "chai";
 import chaiAsPromised from "chai-as-promised";
-import { ServiceBusSender } from "../../src";
-import { ServiceBusReceiver } from "../../src/receivers/receiver";
+import { ServiceBusReceiver, ServiceBusSender } from "../../src";
 import {
-  ServiceBusClientForTests,
-  createServiceBusClientForTests,
   testPeekMsgsLength,
-  EntityName,
-  getRandomTestClientType
+  addServiceBusClientForLiveTesting,
+  AutoGeneratedEntity
 } from "../public/utils/testutils2";
 import { AmqpAnnotatedMessage } from "@azure/core-amqp";
 import { generateUuid } from "@azure/core-http";
+import { TestClientType } from "./utils/testUtils";
 
 const should = chai.should();
 chai.use(chaiAsPromised);
 const assert = chai.assert;
 
-const anyRandomTestClientType = getRandomTestClientType();
-
-let serviceBusClient: ServiceBusClientForTests;
-let entityNames: EntityName;
-let sender: ServiceBusSender;
-let receiver: ServiceBusReceiver;
-const sessionId = "session-1";
-
-function afterEachTest(): Promise<void> {
-  return serviceBusClient.test.afterEach();
-}
-
-function getSampleAmqpAnnotatedMessage(randomTag?: string): AmqpAnnotatedMessage {
-  if (randomTag == null) {
-    randomTag = Math.random().toString();
-  }
-
-  return {
-    body: `message body ${randomTag}`,
-    bodyType: "data",
-    header: {
-      deliveryCount: 10, // TODO: Doesn't make sense to set on the message to be sent, should this be removed for sending?
-      durable: false,
-      firstAcquirer: false,
-      priority: 20,
-      timeToLive: 100000
-    },
-    applicationProperties: {
-      propOne: 1,
-      propTwo: "two",
-      propThree: true,
-      propFour: Date()
-    },
-    // deliveryAnnotations - TODO: should this be removed for sending?
-    footer: {
-      propFooter: "foot"
-    },
-    messageAnnotations: { propMsgAnnotate: "annotation" },
-    properties: {
-      contentEncoding: "application/json; charset=utf-8",
-      correlationId: randomTag,
-      messageId: generateUuid()
-    }
-  };
-}
+[
+  // when we encode messages the partition+session queues are basically the "hardest" one
+  // to do right. There's less concern with the non-session version.
+  // getRandomTestClientTypeWithNoSessions(),
```

Comment:
So we are ditching the "non session" random client then in favor of partitioned/partition+session?

Reply:
Not for everything, but for these, since it's the most intricate, yes.